install/gce-installation-guide.md

```diff
@@ -0,0 +1,119 @@
+
+    diskSizeGb: '10'
+    kind: compute#image
+    licenseCodes:
+    - '1002001'
+    - '5926592092274602096'
+    licenses:
+    - https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx
+    - https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/licenses/ubuntu-1804-lts
+    name: ubuntu-1804-lts-nested
+    sourceImage: https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-1804-bionic-v20180522
+    sourceImageId: '3280575157699667619'
+    sourceType: RAW
+    status: READY
+
+The primary criterion of interest here is the presence of the "enable-vmx" license. Without that licence Kata will not work. The presence of that license instructions the Google Compute Engine hypervisor to enable Intel's VT-x instructions in virtual machines created from the image. Note that nested virtualizaiton is only available in VMs running on Intel Haswell or later CPU microarchitectures.
+
+### Verifying VMX is Enabled
+
+Assuming you created a nested-enabled image using the instructions above you can verify that VMs created from this image are VMX-enabled with the following:
+
+1. Create a VM from the image created above
+
+    ```bash
+    gcloud compute instances create \
+        --image $IMAGE_NAME \
+        --machine-type n1-standard-2 \
+        --min-cpu-platform "Intel Broadwell" \
+        kata-testing
+    ```
+
+> **NOTE**: In most zones the `--min-cpu-platform` argument can be omitted. It is only necessary in GCE Zones that include hosts based on Intel's Ivybridge platform.
+
+2. Verify that it has the VMX CPUID flag set
+
+    ```bash
+    gcloud compute ssh kata-testing
+    
+    lscpu |grep vmx >/dev/null 2>&1 && echo "VMX Enabled! :)" || echo "No VMX :("
+    ```
+
+If this fails, ensure that your instance was created from the correct image and that it has the `enable-vmx` license listed above.
+
+## Installing Kata
+
+From here you can follow the Kata intallation directions for your distribution of choice:
+
+ - [Ubuntu](ubuntu-installation-guide.md)
+ - [CentOS](centos-installation-guide.md)
+ - [Fedora](fedora-installation-guide.md)
+
+## Creating a Kata-enable Image
+
+If, after installing Kata, you would like to create an image to preserve the fruits of your labor, you can do the following:
+
+```bash
+gcloud compute instances stop kata-testing
+gcloud compute images create \
+    --source-disk kata-testing \
+    kata-base
+```
+
+The result should be an image including any changes you'd made to the `kata-testing` instance as well as the `enable-vmx` flag. You can verify this with `gcloud compute images describe kata-base`. The result should be similar to the following (some fields omitted for clarity):
```

Comment:
Suggested rewrite: 

The result is an image that includes any changes made to the `kata-testing` instance as well as the `enable-vmx` flag. Verify this with `gcloud compute images describe kata-base`. The result, which omit some fields for clarity, should be similar to the following:

Reply:
Done.